presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -244,6 +258,12 @@ Approximate Aggregate Functions
     Each element of the array must be between zero and one, and the array must
     be constant for all input rows.
 
+.. function:: approx_percentile(x, w, percentages, accuracy) -> array<[same as x]>
+
+    As ``approx_percentile(x, w, percentages)``, but with a maximum rank error of
+    ``accuracy``. ``accuracy`` must be a value greater than zero and less than
+    one, and it must be constant for all input rows.
```

Comment:
What does it mean for it to be constant for all input rows?

Reply:
I actually copy-pasted this from the other descriptions, but it means this:
The aggregation function actually takes a full set of parameters for each row being aggregated.  Thus the quantity, weight, etc are taken from each row.  But actually, so are the percentile (or percentile array) and accuracy parameters!  However, if in the query, you just put a constant/literal value (like 0.99 for percentile, or 0.01 for accuracy), presto expands this and feeds those constant values in each row.  you could calculate it based on the row!  But that would be confusing and meaningless.

Would it be better to say "accuracy must be a constant value greater than zero and less than one"?

Reply:
I think so because a naive reader like myself couldn't even imagine that you could feed it the output of an expression like say random.